bower.json

```diff
@@ -26,5 +26,8 @@
     "document-register-element": "~0.1.6",
     "flipsnap": "0.3.0",
     "salvattore-moox": "1.0.9"
+  },
+  "resolutions": {
+    "marketplace-core-modules": "1.3.0"
```

Comment:
It's because there is a circular dependency with `marketplace-elements`. I bumped the `marketplace-frontend` patch version and that seems to fix the issue.